assets/src/dashboard/utils/formatBytes.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript/18650828#18650828?newreg=85b17a31d1594ab7bdce436172642949
+const formatBytes = (bytes, decimals = 2) => {
```

Comment:
We should try to avoid copying & adapting code due to potential licensing issues.

Perhaps we could port [the PHP version of this utility from WordPress](https://developer.wordpress.org/reference/functions/size_format/#source) to JavaScript?

Feels more readable too as it doesn't use `log` or `pow`.

Interestingly, the units should be translatable too

Reply:
Sure, didn't think about the units being converted. 

Added `maxUploadFormatted` to dashboard config using `use_size` instead - works as expected. Is this what you had in mind? 
`'maxUploadFormatted' => size_format( $max_upload_size ),`

Reply:
I was more talking about refactoring the JS util but this certainly easier 😄 👍 

Looks great!